### PR TITLE
Add cost calculation tutorial

### PR DESCRIPTION
**Pull request by luiscantero**
* Markdown tutorial
* ARM template to deploy simulated factory
* Update readme
* Cosmetic change to diag info

### CODE REVIEW

File: docs/tutorials/tut-iiot-cost-estimation.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    "HostConfig": {
+        "PortBindings": {
+            "62222/tcp": [
+                {
+                    "HostPort": "62222"
+                }
+            ]
+        },
+        "Binds": [
+            "x509certstores:/root/.dotnet/corefx/cryptography/x509stores",
```
Comment (cristipogacean) [inline docs/tutorials/tut-iiot-cost-estimation.md:79]: do we need this bind?

File: docs/tutorials/tut-iiot-cost-estimation.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        "--mm=PubSub",
+        "--me=Uadp"
+    ],
+    "HostConfig": {
+        "PortBindings": {
+            "62222/tcp": [
+                {
+                    "HostPort": "62222"
+                }
+            ]
```
Comment (cristipogacean) [inline docs/tutorials/tut-iiot-cost-estimation.md:76]: I think we are not using this bind

File: docs/tutorials/tut-iiot-cost-estimation.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+3. On the IoT Hub's blade, add an IoT Edge device
+4. Click on the new IoT Edge device, then *Set Modules -> Add Marketplace Module* and add the following:
+  - OPC Twin
+  - OPC Publisher
+5. Set the container options for the OPC Publisher. The `si` value sets publishing of messages to the IoT Hub to every 10 s. The option `mm` sets the messaging mode and `me` the encoding.   
+```
+{
+    "Hostname": "publisher",
+    "Cmd": [
+        "publisher",
```
Comment (cristipogacean) [inline docs/tutorials/tut-iiot-cost-estimation.md:58]: this is not needed

File: docs/tutorials/tut-iiot-cost-estimation.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        "--pf=/appdata/pn.json",
+        "--di=60",
+        "--to",
+        "--aa",
+        "--si=10",
+        "--bs=100",
+        "--fd=true",
+        "--fm=true",
```
Comment (cristipogacean) [inline docs/tutorials/tut-iiot-cost-estimation.md:66]: both --fd and --fm are irrelevant for these tests, --fd is much better to be set to false for load tests, so my recommendation is to remove both flags

File: docs/tutorials/tut-iiot-cost-estimation.md (hunk 1 above)
Comment (luiscantero) [reply docs/tutorials/tut-iiot-cost-estimation.md]: Removed

File: docs/tutorials/tut-iiot-cost-estimation.md (hunk 2 above)
Comment (luiscantero) [reply docs/tutorials/tut-iiot-cost-estimation.md]: Removed

File: docs/tutorials/tut-iiot-cost-estimation.md (hunk 4 above)
Comment (luiscantero) [reply docs/tutorials/tut-iiot-cost-estimation.md]: Removed

File: docs/tutorials/tut-iiot-cost-estimation.md (hunk 3 above)
Comment (luiscantero) [reply docs/tutorials/tut-iiot-cost-estimation.md:57]: Removed

File: docs/tutorials/tut-iiot-cost-estimation.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+However, if every device instead sends a 4-KB message every 4 seconds, the messages will be charged as 10 message chunks every 4 seconds. This translates to 86,400 / 4 = 21,600 messages per device, for a total of 216,000 messages for all 10 devices per day. In this case, one S1 unit would be enough.
+
+If the requirements state that waiting 4 seconds for the data is not acceptable, that will of course affect the monthly costs. See [IoT Hub quotas and throttling](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-quotas-throttling) and [IoT Hub limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#iot-hub-limits) for further relevant information.
+
+### Reference factories
+The following factory setups are provided for reference. A slow node changes every 10 s, a fast one every 1 s.
+
+| Factory size | PLCs | Slow nodes | Fast nodes | Values/s/PLC | VM size | IoT Hub |
+| ------------ | ---: | ---------: | ---------: | -----------: | ------- | ------- |
+| Small	       |   25 |        250 |         50 |           75 | >= B2ms | 8 * S1  |
```
Comment (cristipogacean) [inline docs/tutorials/tut-iiot-cost-estimation.md:38]: is here a tab/space too much?

Comment (luiscantero) [reply docs/tutorials/tut-iiot-cost-estimation.md:38]: No, this renders fine in VS Code. Must be a GH issue.
image

**Pull request merged**